nginx/config.go

```diff
@@ -224,16 +224,38 @@ http {
 )
 
 func WriteCerts(routerConfig *model.RouterConfig, sslPath string) error {
+	// Start by deleting all certs and their corresponding keys. This will ensure certs we no longer
+	// need are deleted. Certs that are still needed will simply be re-written.
+	allCertsGlob, err := filepath.Glob(filepath.Join(sslPath, "*.crt"))
+	if err != nil {
+		return err
+	}
+	allKeysGlob, err := filepath.Glob(filepath.Join(sslPath, "*.key"))
+	if err != nil {
+		return err
+	}
+	for _, cert := range allCertsGlob {
+		err = os.Remove(cert)
```

Comment:
Fixing it. It's probably a matter of personal style that I rarely collapse them like that. Probably because 9 times out of 10, I'm getting multiple return values from a function call and have big plans for that other variable. But in this case, it can be done, so I will.